app/src/main/res/values-ta/strings.xml

```diff
@@ -1,29 +1,39 @@
-    <string name="save_as_json">JSON ஆக சேமிக்கவும்</string>
-    <string name="new_item">புதிய பொருள்</string>
-    <string name="close">மூடு</string>
-    <string name="value">மதிப்பு</string>
-    <string name="name">பெயர்</string>
-    <string name="saved">சேமிக்கப்பட்டது</string>
-    <string name="failed">வேலை செய்யவில்லை</string>
-    <string name="saved_to_file">%s இல் சேமிக்கப்பட்டது</string>
-    <string name="theme_system">முறையைப் பின்பற்றவும்</string>
-    <string name="theme_day">நாள்</string>
-    <string name="theme_night">இரவு</string>
-    <string name="theme_battery">பேட்டரி சேமிப்பாளரைப் பின்பற்றவும்</string>
-    <string name="theme">தீம்</string>
+    <string name="save">Save</string>
+    <string name="error_no_support"><b>SetEdit</b> does not currently support editing this table. To enable editing, run the following command in a root or ADB shell:</string>
+    <string name="error_unexpected"><b>Unexpected failure:</b></string>
+    <string name="delete">Delete</string>
+    <string name="help">Get help</string>
+    <string name="system_table">System Table</string>
+    <string name="secure_table">Secure Table</string>
+    <string name="global_table">Global Table</string>
+    <string name="android_properties">Android Properties</string>
+    <string name="java_properties">Java Properties</string>
+    <string name="linux_environment">Linux Environments</string>
+    <string name="startup_warning"><b>Warning!</b> We do not guarantee that problems caused by the improper use of this utility can be fixed, and we are unable to help you with any such problems. We support only this app, not your device.</string>
+    <string name="save_as_json">Save as JSON</string>
+    <string name="new_item">New Item</string>
+    <string name="close">Close</string>
+    <string name="value">Value</string>
+    <string name="name">Name</string>
+    <string name="saved">Saved</string>
+    <string name="failed">Failed</string>
+    <string name="saved_to_file">Saved to %s</string>
+    <string name="theme_system">Follow system</string>
+    <string name="theme_day">Day</string>
+    <string name="theme_night">Night</string>
+    <string name="theme_battery">Follow battery saver</string>
+    <string name="theme">Theme</string>
+    <string name="shortcut">Shortcut</string>
+    <string name="icon_picker">Icon Picker</string>
+    <string name="create_or_append_to_an_existing_shortcut_rather_than_apply">Create or append to an existing shortcut rather than applying it</string>
+    <string name="edit_shortcuts">Edit shortcuts</string>
+    <string name="grant_secure_setting">Grant secure setting</string>
+    <string name="append">Append</string>
+    <string name="disable">Disable</string>
+    <string name="enable">Enable</string>
+    <string name="disabled">Disabled</string>
+    <string name="delete_action">Delete Action</string>
+    <string name="cancel">Cancel</string>
```

Comment:
This file is for Tamil language, not English. Leave it as it was.

Reply:
You are right again and I'm sorry for the mess.